dll/win32/iernonce/registry.cpp

```diff
@@ -311,8 +311,11 @@ BOOL RunOnceExInstance::Exec(_In_opt_ HWND hwnd)
         m_SectionList[i].CloseAndDelete(m_RegKey);
     }
 
-    m_RegKey.DeleteValue(L"Title");
-    m_RegKey.DeleteValue(L"Flags");
+    if (m_RegKey)
```

Comment:
Could you make a separate commit or PR for this? (this is unrelated to DEBUG stuff)